packages/wallet/src/utils/contract-utils.ts

```diff
@@ -1,18 +1,21 @@
+import {State, validTransition} from "@statechannels/nitro-protocol";
+// @ts-ignore this is not supposed to exist at the prepare step
```

Comment:
I don't understand this comment.

Reply:
Removed - the message behind it wasn't factual anyway.